core/src/main/java/io/questdb/cairo/GeoHashes.java

```diff
@@ -193,17 +162,23 @@ public static long fromString(CharSequence hash, int start, int end) throws Nume
         return geohash;
     }
 
-    private static long appendChar(long geohash, char c) throws NumericException {
-        byte idx;
+    private static long appendChar(long hash, char c) throws NumericException {
         if (c >= 48 && c < 123) { // 123 = base32Indexes.length + 48
-            idx = base32Indexes[c - 48];
+            byte idx = base32Indexes[c - 48];
             if (idx >= 0) {
-                return (geohash << 5) | idx;
+                return (hash << 5) | idx;
             }
         }
         throw NumericException.INSTANCE;
     }
 
+    public static byte encodeChar(char c) throws NumericException {
```

Comment:
method not called from anywhere/not tested

Reply:
This method is called and tested from row copier assembler, IntelliJ cannot work this out. I will add explicit tests for this method though to remove the warning, thanks!

Reply:
👍